integration-tests/load/automationv2_1/automationv2_1_test.go

```diff
@@ -124,6 +132,35 @@ Password = '%s'`
 	}
 )
 
+func setUpDataStreamsWireMock(url string) error {
+	wm := gowiremock.NewClient(url)
+	rule200 := gowiremock.Get(gowiremock.URLPathEqualTo("/api/v1/reports/bulk")).
+		WithQueryParam("feedIDs", gowiremock.EqualTo("0x000200")).
+		WillReturnResponse(gowiremock.NewResponse().
+			WithBody(`{"reports":[{"feedID":"0x000200","validFromTimestamp":0,"observationsTimestamp":0,"fullReport":"0x000abc"}]}`).
+			WithStatus(200))
+	err := wm.StubFor(rule200)
+	if err != nil {
+		return err
+	}
+	resp, err := http.Post(fmt.Sprintf("%s/__admin/mappings/save", url), "application/json", nil)
+	if err != nil {
+		return errors.New("error saving wiremock mappings")
+	}
+	defer func(Body io.ReadCloser) {
```

Comment:
I would just ignore the error returned by closing the Body, so I'd just lose this closure and replace it with a single line `defer resp.Body.Close()`; hopefully we don't have annyoing linters that would complain about that. If we do, I'd explicitly discard the error with `defer func() { _ = resp.Body.Close() }()` Since I don't think we should panic if we can't close the body